packages/framework/src/Twig/HiddenPriceExtension.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Twig;
+
+use Shopsys\FrameworkBundle\Model\Customer\User\CustomerUser;
+use Shopsys\FrontendApiBundle\Component\Price\MoneyFormatterHelper;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+class HiddenPriceExtension extends AbstractExtension
+{
+    /**
+     * @return array
+     */
+    public function getFilters(): array
+    {
+        return [
+            new TwigFilter(
+                'hidePrice',
+                $this->hidePriceFilter(...),
+            ),
+        ];
+    }
+
+    /**
+     * @param string $price
+     * @param \Shopsys\FrameworkBundle\Model\Customer\User\CustomerUser|null $customerUser
+     * @return string
+     */
+    public function hidePriceFilter(string $price, ?CustomerUser $customerUser): string
+    {
+        if ($customerUser !== null && !$customerUser->canSeePrices()) {
```

Comment:
wouldn't it be better to use
```php
$this->security->isGranted(CustomerUserRole::ROLE_API_CUSTOMER_SEES_PRICES)
```
?

Reply:
I think it will not cover all scenarios. e.g. updating order status from 3rd party, there is no active user. 
What do you think ?

Reply:
Oh, I'm sorry, I thought this
``` 
$customerUser !== null && $this->security->isGranted(CustomerUserRole::ROLE_API_CUSTOMER_SEES_PRICES)
```